### ISSUE TITLE
Enable ArgoCD syncing on cloud

### ISSUE THREAD

**Issue by daxmc99** [sourcegraph/sourcegraph#19369]
From https://docs.google.com/document/d/1IohtPWJicyg-0sydCblO-25JHm3LwP0yIxopI4x-AqQ/edit#bookmark=id.whceth6oj3v1 
Tasks:
- [x] Breakout every application into a separate Application
- [x] Have every application track a relevant git tag `deploy/${DEPLOYMENT_NAME}` 
- [ ] Disable existing CD pipeline
- [ ] Enable ArgoCD auto-sync
- [x] Setup Ingress
- [x] Create a domain for ArgoCD

**Comment by ggilmore** [comment]
@ggilmore add separate issue for pairing to add dhall based image updater pipeline  (.5 day)

**Comment by davejrt** [comment]
> @ggilmore add separate issue for pairing to add dhall based image updater pipeline (.5 day)

would be interested in pairing on this dhall based work @ggilmore. Please feel free to assign me to the pairing task

**Comment by bobheadxi** [comment]
Probably requires https://github.com/sourcegraph/sourcegraph/issues/19228?

**Comment by daxmc99** [comment]
Blocked by https://github.com/sourcegraph/sourcegraph/issues/27036
